dotcom-rendering/src/model/enhanceCollections.ts

```diff
@@ -39,6 +39,12 @@ const decideSubLinkFormat = ({
 	return linkFormat;
 };
 
+// There are certain collections used to add things like global styles to a front page
+// which we don't want to use on DCR as we'd like to try and migrate these styles into DCR itself.
+const blockedCollections = [
+	'ba744122-6c99-417f-8ddd-5a5905eb4928', // Palette styles new do not delete
+];
```

Comment:
Why do we need to block this? Is the css in here causing issues or is this a drive by?

Reply:
If we're disabling JS and preventing CSS from breaking out of snaps I don't think this is needed anymore. This was to stop things like the sombre pallet snap that purely exists to change other elements on the page from loading.